Plot_Data.py

```diff
@@ -0,0 +1,56 @@
+import Use_Data as data
+import matplotlib.pyplot as plt
+from pathlib import Path
+
```

Comment:
```suggestion
from matplotlib import rc
rc('text', usetex=True)

```

Added this incase we want to add latex formatting in the future